proto/rill/runtime/v1/queries.proto

```diff
@@ -342,7 +344,9 @@ message MetricsViewComparisonRequest {
   repeated MetricsViewComparisonSort sort = 5;
   TimeRange time_range = 6;
   TimeRange comparison_time_range = 7;
-  MetricsViewFilter filter = 8;
+  Condition where = 8;
+  Condition having = 14;
```

Comment:
Consider using `Expression` directly, simpler to have one entrypoint (and matches the name of the file). Also slightly more flexible in that you can put something like just `WHERE true`